src/modules/10-common/hooks/__tests__/useEventSourceListener.test.tsx

```diff
@@ -109,4 +122,16 @@ describe('use Event Source Stream', () => {
     expect(addEventListener).toHaveBeenCalledTimes(0)
     expect(removeEventListener).toHaveBeenCalledTimes(3)
   })
+
+  test('Custom headers should be passed', async () => {
+    render(
+      <TestWrapper path={routes.toProjects({ accountId: 'dummy' })} pathParams={{ accountId: 'dummy' }}>
+        <Wrapped lazy={true} headers={{ 'x-foo': 'bar' }} />
+      </TestWrapper>
+    )
+
+    expect(eventSourcePolyfillFn).toHaveBeenCalled()
+    expect(receivedHeaders['x-foo']).toEqual('bar')
+    expect(receivedHeaders['Authorization']).toEqual('Bearer token')
```

Comment:
Just an FYI, but you can use `toHaveProperty` to assert against objects.

```suggestion
     expect(receivedHeaders).toHaveProperty('x-foo', 'bar')
     expect(receivedHeaders).toHaveProperty('Authorization', 'Bearer token')
```

Reply:
thats cleaner! thanks.